### PR TITLE
Address review comments left over from #876

### PR DESCRIPTION
**Pull request by marc-casperlabs**

### CODE REVIEW

File: node/src/components/networking_metrics.rs [hunk 1]
```diff
@@ -22,15 +22,13 @@ impl NetworkingMetrics {
...
             "net_direct_message_requests",
-            "How often a request to send a message directly to a peer was made.",
-        )?;
-        let open_connections = IntGauge::new(
-            "net_open_connnections",
-            "Current number of open connections.",
+            "number of requests to send a message directly to a peer",
         )?;
+        let open_connections =
+            IntGauge::new("net_open_connnections", "number of established connections")?;
```
Comment (afck) [inline node/src/components/networking_metrics.rs:28]:
```suggestion
            IntGauge::new("net_open_connections", "number of established connections")?;
```

**Review by afck — APPROVED** [review]
One more typo. :detective:

**Review by Fraser999 — APPROVED** [review]

**Review by EdHastingsCasperLabs — APPROVED** [review]

**Comment by goral09** [comment]
Can we merge this?

**Pull request merged**